Sources/Rendering/OpenGL/VolumeMapper/index.js

```diff
@@ -1682,6 +1687,75 @@ function vtkOpenGLVolumeMapper(publicAPI, model) {
 
     model.VBOBuildTime.modified();
   };
+
+  publicAPI.updateLabelOutlineThicknessTexture = (volume) => {
+    const labelOutlineThicknessArray = volume
+      .getProperty()
+      .getLabelOutlineThickness();
+
+    if (!labelOutlineThicknessArray) {
+      return;
```

Comment:
This means that if you reset an existing array, it won't be removed from the GPU.
Shouldn't it be released from GPU memory?

What behavior do you expect in the shader ?

Reply:
it is not about reset I think, it is either it is defined or not. I tried changing it and it worked fine

https://github.com/Kitware/vtk-js/assets/7490180/4dc6d881-2a83-4073-bf94-2292fb78aaea

Reply:
When you start you have an array, what happens if you call: `setLabelOutlineThicknessArray(null)` ?

Reply:
removed that unnecessary check and let the toString handle it